package.json

```diff
@@ -44,6 +44,8 @@
   "devDependencies": {
     "babel-core": "6.6.0",
     "babel-eslint": "5.0.0",
+    "babel-istanbul": "^0.6.0",
```

Comment:
I've been trying to use exact requirements (``"babel-istanbul": "0.6.0"``) so that there are no surprises when another contributor installs a fresh copy of the repo. Theoretically the caret bug fix releases won't introduce breaking changes but you have to trust that all other developers will be good about that.